NEWS.md

```diff
@@ -1,6 +1,9 @@
 Hit tab to unselect buttons and scroll through the text using UP/DOWN or PGUP/PGDN.
 All announcements are stored in `/opt/openhabian/docs/CHANGELOG.md` for you to lookup.
 
+## new config option initialconfig ## May 8, 2021
```

Comment:
Capitalization is your friend. 

``` suggestion
## New `openhabian.conf` option `initialconfig` ## May 8, 2021
```

Reply:
and I had to change the date, damn it ...

Reply:
addressed all of your points need to get some sleep now. Merge at will.